internal/log/logger.go

```diff
@@ -47,6 +47,17 @@ func New(level Level, sink Sink) *Logger {
 func (l Logger) Scope(scope string) *Logger {
 	return l.Attrs(map[string]string{scopeKey: scope})
 }
+
+func (l Logger) AppendScope(scope string) *Logger {
```

Comment:
Setting scope in pubsub was losing runner scope which made pubsub logs harder to follow